contracts/hackatom/src/contract.rs

```diff
@@ -53,8 +65,17 @@ pub fn init<S: Storage, A: Api>(
 pub fn handle<S: Storage, A: Api>(
     deps: &mut Extern<S, A>,
     params: Params,
-    _: HandleMsg,
+    msg: HandleMsg,
 ) -> Result<Response> {
+    match msg {
+        HandleMsg::Release {} => try_release(deps, params),
+        HandleMsg::CpuLoop {} => cpu_loop(),
+        HandleMsg::StorageLoop {} => storage_loop(deps),
+        HandleMsg::Panic {} => do_panic(),
```

Comment:
Is there a reason why some of those are `try_` prefixed? This looks random.

Reply:
Panic has `do_` as `panic` is a reserved keyword.

The other two have no try, as they will always loop.
I had the `try_`, as it does a few checks (eg. authorization) before releasing, hence the `try_`.

But yeah, maybe names are a bit too different

Reply:
I normalized them a bit